garage/tf/policies/categorical_conv_policy_with_model.py

```diff
@@ -0,0 +1,139 @@
+            dimension (5 x 5).
+        conv_filters(tuple[int]): Number of filters. For example, (3, 32) means
+            there are two convolutional layers. The filter for the first layer
+            has 3 channels and the second one with 32 channels.
+        conv_strides(tuple[int]): The stride of the sliding window. For
+            example, (1, 2) means there are two convolutional layers. The
+            stride of the filter for first layer is 1 and that of the second
+            layer is 2.
+        conv_pad: The type of padding algorithm to use,
+            either 'SAME' or 'VALID'.
+        name: Variable scope of the policy.
+        hidden_sizes: Output dimension of dense layer(s).
+        hidden_nonlinearity: Activation function for
+                    intermediate dense layer(s).
+        output_nonlinearity: Activation function for
+                    output dense layer.
+        layer_normalization: Bool for using layer normalization or not.
+
+    """
+
+    def __init__(self,
+                 env_spec,
+                 conv_filters,
+                 conv_filter_sizes,
+                 conv_strides,
+                 conv_pad,
+                 name='CategoricalConvPolicy',
+                 hidden_sizes=[],
+                 hidden_nonlinearity=tf.nn.relu,
+                 output_nonlinearity=tf.nn.softmax,
+                 layer_normalization=False):
+        assert isinstance(env_spec.action_space, Discrete), (
+            'CategoricalConvPolicy only works with akro.tf.Discrete'
+            'action space.')
+        super().__init__(name, env_spec)
+        self.obs_dim = env_spec.observation_space.shape
+        self.action_dim = env_spec.action_space.n
+
+        self.model = Sequential(
+            CNNModel(
+                filter_dims=conv_filter_sizes,
+                num_filters=conv_filters,
+                strides=conv_strides,
+                padding=conv_pad,
+                hidden_nonlinearity=hidden_nonlinearity,
+                name='CNNModel'),
+            MLPModel(
+                output_dim=self.action_dim,
+                hidden_sizes=hidden_sizes,
+                hidden_nonlinearity=hidden_nonlinearity,
+                output_nonlinearity=output_nonlinearity,
+                layer_normalization=layer_normalization,
+                name='MLPModel'))
+
+        self._initialize()
+
+    def _initialize(self):
+        state_input = tf.placeholder(tf.float32, shape=(None, ) + self.obs_dim)
+
+        with tf.variable_scope(self._variable_scope):
+            self.model.build(state_input)
+
+        self._f_prob = tf.get_default_session().make_callable(
+            self.model.outputs, feed_list=[self.model.input])
+
+    @property
+    def vectorized(self):
+        """Vectorized or not."""
+        return True
+
+    @overrides
+    def dist_info_sym(self, obs_var, state_info_vars=None, name=None):
+        """Symbolic graph of the distribution."""
```

Comment:
please provide full docstrings for all methods (unless the parent class provides a docstring which is equivalent)